fe/fe-core/src/main/java/org/apache/doris/planner/PlanNode.java

```diff
@@ -175,26 +176,32 @@ protected PlanNode(PlanNodeId id, PlanNode node, String planNodeName) {
                 "V" + planNodeName : planNodeName;
         this.numInstances = 1;
         this.nodeType = node.getNodeType();
-        this.statsDeriveResult.set(node.getStatsDeriveResult());
+        this.statsDeriveResult = new StatsDeriveResult(
```

Comment:
It should not be set directly in the constructor.

Reply:
When the derivation of statistics for all nodes is complete, the initialization needs to be removed.
The reason for the current increase is that some uts cannot pass